Firestore/core/test/unit/local/lru_garbage_collector_test.cc

```diff
@@ -657,7 +660,9 @@ TEST_P(LruGarbageCollectorTest, RemoveTargetsThenGC) {
 TEST_P(LruGarbageCollectorTest, GetsSize) {
   NewTestResources();
 
-  int64_t initial_size = gc_->CalculateByteSize();
+  StatusOr<int64_t> maybe_initial_size = gc_->CalculateByteSize();
```

Comment:
It's probably too much trouble to test the failure case?

Reply:
Under the current code structure, the test would have to intentionally create a race condition to prompt the failure. Given how quickly you can stat files I'm not sure that's a condition we can create reliably.

I could change this around so that the underlying size calculation takes an iterator as an argument, but this couldn't be expressed in the platform agnostic code this is testing and so it would only be a low-level unit test that errors during iteration are actually propagated into the result. I'm not sure this is terribly useful though.

All told, I'm in the camp of more trouble than it's worth without dramatically altering the dependency structure of all these components.

Reply:
Sounds good to me.